src/lib/api.ts

```diff
@@ -0,0 +1,14 @@
+export const API_URL = process.env.PROMOCHATOR_API ?? "";
```

Comment:
**suggestion:** I think it would be wise to check whether we have a API_URL set in env variables to be 100% sure that we'll avoid errors related to it not being there. Either this or we can default this to `[REDACTED]` instead of an empty string.

**note:** Remember we need the `https://` prefix

Reply:
my point there was to not expose the url - probably completely useless because if someone wants to reveal what's the backend url then they will do that analyzing reqs/resps

More explicit solution came to my mind:
```suggestion
export const API_URL = process.env.PROMOCHATOR_API ?? "";

if (API_URL === "") {
  throw new Error("API_URL was not set in enviroment variables!");
} ;
```

this solution requires adding `src/instrumentation.ts` which should look like this:
```
export async function register() {
  await import("@/lib/api");
}
```

And it throws an error when running server in dev/prod mode.

What do you think? @maciejkrol18

Reply:
Looks good, we can use your proposed solution